frontend/src/components/templates/Form/index.js

```diff
@@ -158,7 +159,17 @@ export const Form = ({
         >
           <HeadSection />
           <StepperSection />
-          {children}
+          <HideSectionsContainer
+            SubmissionPanel={() => (
```

Comment:
This code is equivalent to creating a functional component "on the fly" and passing it as a prop, delegating the responsibility of instantiation to `HideSectionsContainer`.
This will not work as this callback will be different on every render and will create a new component.

Remember that the code between jsx tags (children), is in fact just another prop.
You can use your own props to pass "other children" or "slots".

Also props should be in camelCase

Replace this code with this implementation and change HideSectionsContainer accordingly

```tsx
<HideSectionsContainer
            submissionPanel={
              <SubmissionPanel
                enrollment={enrollment}
                handlePostEvent={handlePostEvent}
                updateEnrollment={dispatchSetEnrollment}
              />
            }
          >
 ```